react-app/ccf/src/pages/forgot-password/ForgotPassword.css

```
@@ -1,143 +1,175 @@
-    font-family: 'Roboto', sans-serif;
-    gap: 20px;
-    width: 80vw;
-    padding-left: 0 !important;
-  }
+.forgot-content {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+  font-family: 'Roboto', sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)